source/_patterns/01-base/02-html-elements/14-paragraph/_paragraph.scss

```diff
@@ -2,7 +2,8 @@
 // Paragraph styles.
 
 p {
-  margin: 0 0 rem($vertical-spacing);
+  @include display-text-style(body);
```

Comment:
should this need to be explicitly stated on the p tag?  shouldn't it be inherited?